examples/embeddable_examples/public/app/register_embeddable.tsx

```diff
@@ -36,17 +41,64 @@ export const RegisterEmbeddable = () => {
           Run the example embeddables by creating a dashboard, clicking <em>Add panel</em> button,
           and then selecting <em>Embeddable examples</em> group.
         </p>
+      </EuiText>
+
+      <EuiSpacer size="l" />
+
+      <EuiText>
+        <h2>Show embeddables in the Add panel menu</h2>
         <p>
           Add your own embeddables to <em>Add panel</em> menu by attaching an action to the{' '}
           <strong>ADD_PANEL_TRIGGER</strong> trigger. Notice usage of <strong>grouping</strong> to
           nest related panel types and avoid bloating <em>Add panel</em> menu. Please reach out to
           @elastic/kibana-presentation team to coordinate menu updates.
         </p>
       </EuiText>
-
+      <EuiSpacer size="s" />
       <EuiCodeBlock language="jsx" fontSize="m" paddingSize="m">
         {registerAttachActionSource}
       </EuiCodeBlock>
+
+      <EuiSpacer size="l" />
+
+      <EuiText>
+        <h2>Specify panel placement settings (optional)</h2>
+        <p>
+          Embeddable factories can specify placement settings for their embeddables. This is done by
+          registering an embeddable type with a <em>getPanelPlacementSettings</em> function using{' '}
+          <strong>registerDashboardPanelPlacementSetting</strong>. The{' '}
+          <em>getPanelPlacementSettings</em> function that takes the serialized state of the
+          embeddable and returns an object with optional width, height, and strategy properties. The
+          width and height properties are numbers that specify the dimensions of the panel in the
+          Dashboard where the max width is <EuiCode>48</EuiCode> and the max height is virtually
+          unlimited. The strategy property is an enum that specifies the placement strategy to use.
+          The example below shows how the Field List embeddable developer example specificies its
+          placement settings on a Dashboard.
+        </p>
+      </EuiText>
+      <EuiSpacer size="s" />
+
+      <EuiCodeBlock language="jsx" fontSize="m" paddingSize="m">
```

Comment:
How about changing the example to only show `registerDashboardPanelPlacementSetting` to keep the example focused on the text description.